packages/paste-website/src/pages/foundations/elevation/index.mdx

```diff
@@ -0,0 +1,218 @@
+
+<ResponsiveImage src={Level2AccentChange} alt="" />
+
+<ApplyLevel2ImgCaption />
+
+2. **Use a soft and sharp shadow** for elements that have temporary prominence or subtly float above the content.
+
+*Interactive states like vertical or horizontal scroll can dynamically use a soft, sharp shadow to indicate that there is overflown content.
+
+<ResponsiveImage src={Level2RisenShadow} alt="" />
+
+<ApplyLevel2ImgCaption />
+
+### Level 3
+
+Elements that rest at this level, float above all other levels, and tend to contain crucial actions or information that temporarily becomes the top hierarchy of the surface.
+
+1. **Use a large, soft shadow with edges**, for elements that are at the top priority and require user focus, but still allow the user to interact with the surrounding content.
+
+For elements with decorative or semantic use of color (background or borders), use a large soft shadow without edges.
+
+
+<ResponsiveImage src={Level3RisenShadow} alt="" />
+
+2. **Use a large soft shadow along with an overlay** for elements that completely isolate the interface to draw full focus from the user.
+
+<ResponsiveImage src={Level3Overlay} alt="" />
+
+## Elevated background color tokens
+
+Subtle tonal shifts are an effective way to communicate hierarchy and prominence for elevated elements without adding visual clutter to the screen.
+
+Paste's elevated background color tokens are derived from a base background color with tonal shifts to indicate elevation. Elevated surfaces darken on light backgrounds and lighten on dark backgrounds, ensuring clarity and consistency across themes.
+
+<LivePreview
+  scope={{
+    Box,
+    Text,
+  }}
+  noInline
+  showOverflow
+>
+  {ElevatedBackgroundColorTokensExample}
+</LivePreview>
+
+Elevated elements across levels that share the same background color as the main page body should use the token `$color-background-weakest`. This token matches the page body tone in light mode but appears slightly lighter in dark mode to subtly reinforce elevation.
+
+<ResponsiveImage src={ElevatedBackgroundTokens} alt="" />
+
+View our full list of Elevated background color tokens
+
+<Callout variant="neutral" marginY="space70">
+  <CalloutHeading as="h4">Different uses of color</CalloutHeading>
+  <CalloutText>
+   <CalloutList as="ul">
+      <CalloutListItem>Elevation Colors: Subtle tones for prominence and hierarchy, used to separate elements visually from the background.</CalloutListItem>
+      <CalloutListItem>Semantic Colors: Used to communicate meaning or status, such as success, warning, or error.</CalloutListItem>
+      <CalloutListItem>Decorative Colors: Reflect the brand identity or give a branded feel to a composition.</CalloutListItem>
+    </CalloutList>
```

Comment:
nit: can you add some spacing between the last list item and the following text? seems squished